src/Microsoft.AspNetCore.SignalR/Proxies.cs

```diff
@@ -54,12 +55,28 @@ public Task InvokeAsync(string method, params object[] args)
         }
     }
 
+    public class AllClientsExceptProxy<THub> : IClientProxy
+    {
+        private readonly HubLifetimeManager<THub> _lifetimeManager;
+        private readonly List<string> _excludedIds;
+
+        public AllClientsExceptProxy(HubLifetimeManager<THub> lifetimeManager, List<string> excludedIds)
```

Comment:
Why not `ICollection<string>`? It is the lowest level interface that offers `Contains()`.
Performance-wise, the dev can choose between sending a List<> or a Set<> or something else.